web-wallet/src/lib/dusk/components/Toast/store.js

```diff
@@ -34,7 +34,7 @@ function addToast (type, message, icon) {
 
 /**
  * @param {string} id
- * @param {NodeJS.Timeout} timeout
+ * @param {ReturnType<typeof setTimeout>} timeout
```

Comment:
This can be just `number` as it is in the browser. 
We need to call `setTimeout` & co. with `window.setTimeout` though to make TS understand.